pkg/autoscaler/statforwarder/forwarder_test.go

```diff
@@ -440,22 +439,14 @@ func TestProcess(t *testing.T) {
 		t.Fatalf("Timeout waiting f.processors got updated")
 	}
 
-	forwardCount := 0
-	// Override the proc so we do not actually send via WebSocket.
-	f.processors[bucket2].proc = func(sm asmetrics.StatMessage) {
-		forwardCount++
-	}
-
+	// Stat1 should be accepted and stat2 should be forwarded.
 	f.Process(stat1)
 	f.Process(stat2)
```

Comment:
Do we call stat2 for checking it won't change its mind?
Do we want to call stat1 twice then to check it's processed twice?

Reply:
1. Yes
2. Added.